spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -259,9 +259,15 @@ def index
     end
 
     context 'when SP requires PIV/CAC' do
+      let(:service_provider) { create(:service_provider) }
+
       before do
         stub_sign_in(user)
-        controller.session[:sp] = { phishing_resistant_requeste: true, piv_cac_requested: true }
+        controller.session[:sp] = {
+          phishing_resistant_requeste: true,
```

Comment:
I see that this was like this before, but does this misspelling ("requeste") affect anything?

Reply:
LOL - removed it and it still passes. Done in 35df66c25